contrib/nushell/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "nushell"
+pkgver = "0.95.0"
+pkgrel = 0
+build_style = "cargo"
+make_build_env = {
+    "PREFIX": "/usr",
+}
+make_install_env = make_build_env
```

Comment:
If we don't explicitly set this env var during compile time, nushell will use `/usr/local` as a prefix, resulting in autoloading of completions happening from `/usr/local/share/nushell/vendor/autoload` instead of `/usr/share/nushell/vendor/autoload`

Reply:
you can just make it `make_env` single variable